src/eeg_series.jl

```diff
@@ -285,15 +240,16 @@ function interctive_toposeries(interactive_scatter, single_topoplot)
     end
 end
 
-function create_axis_options(xlim_topo, ylim_topo)
+function create_axis_options()
```

Comment:
I guess all of these could in principle live in plotConfig?

Reply:
I also had this though, i just was not sure how to implement that